examples/.explainer-config.toml

```diff
@@ -0,0 +1,9 @@
+# Any of these config options can or can not be provided
+# Feel free to place this file in your home directory, i.e: cp examples/.explainer-config.toml ~

```

Comment:
these comments should be removed and just update the `README.md` instead.

Reply:
updated